README.md

```diff
@@ -31,10 +43,15 @@ None
 Example Playbook
 ----------------
 
-```yaml
+```
 - hosts: reboot-servers
   roles:
-    - role: oasis-roles.reboot
+    - role: reboot
```

Comment:
The oasis-roles galaxy namespace should be left in here for the example as users of the role will import via galaxy as `oasis-roles.reboot`